subprojects/packagefiles/json-c/meson.build

```diff
@@ -81,6 +83,7 @@ libjson_c = library(
   sources,
   include_directories: inc,
   install: true,
+  version: '5.4.0',
```

Comment:
For anyone else looking into this. `5.4.0` _is_ the correct version number, or at least the one used in the CMake files.